examples/devicemodel.c

```diff
@@ -56,7 +56,8 @@ static const struct dev_pm_ops devicemodel_pm_ops = {
     .poweroff = devicemodel_suspend,
     .freeze = devicemodel_suspend,
     .thaw = devicemodel_resume,
-    .restore = devicemodel_resume};
+    .restore = devicemodel_resume
```

Comment:
You have to specify the comma at the of this line. That is,
```c
.restore = devicemodel_resume,
```

Then, `clang-format` would indent as expected. Always run `clang-format -i` before submitting.